test/controllers/users_controller_test.rb

```diff
@@ -101,14 +101,15 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
   end
 
   describe 'create method' do
-    it 'should create a user' do
+    it 'should create a user with the role cm' do
       attributes = attributes_for(:user)
-      attributes.delete :role
       assert_difference 'User.count', 1 do
         post users_path, params: { user: attributes }
       end
+      assert(User.last.role == :cm)
```

Comment:
Appreciate the attention to detail on our existing tests!

Minitest makes these nice semantic checks available, and we generally follow them since they do make it a little easier to read as plain english. Thoughts about this?

```suggestion
      assert_equal :cm, User.last.role
```